### PR TITLE
Feature/analytics events

### PR DESCRIPTION
**Pull request by scottusrobus**
Add internal docs for analytic events.

### CODE REVIEW

**Comment by ksheurs** [comment]
Nice @scottusrobus !!

File: source/javascripts/app/_internal.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+(function(window, $) {
+  $(document).on('ready', function() {
+    if (window.location.search.match(/internal\=1/)) {
+      console.log("boom");
```
Comment (ksheurs) [inline source/javascripts/app/_internal.js:4]: fyi console.log

File: source/includes/_resource.analytics.md [hunk 2]
```diff
@@ -3,49 +3,454 @@
...
+
+  // fire when the event happens
+  trackEvent();
+
+  // in the URL property=value will be replaced by all the properties
+  // for the particular event (see to your left)
+```
+
+<section class="text-2 contain is-internal">
+Analytics usage and engagement data stems from from specific events that are tracked based on user activity. These events can be tracked by making a <code>GET</code> request with a series of query parameters (for events properties) to our collector endpoint, shown to the right. The endpoint is a publicly accessible <code>events.gif</code> to work around cross domain issues of making post requests.
```
Comment (ksheurs) [inline source/includes/_resource.analytics.md:35]: customer activity (not user)

File: source/includes/_resource.analytics.md [hunk 3]
```diff
@@ -3,49 +3,454 @@
...
+    </tr>
+    <tr class="text-2 border-bottom border--light-gray">
+      <td><strong class="is-block">ended</strong></td>
+      <td>Playback has stopped due to end of video.</td>
+    </tr>
+  </tbody>
+</table>
 
+<section class="text-2 contain is-internal margin-bottom-medium padding-top-large">
+In addition, video playback triggered events should be sent with the follow properties as query parameters in the <code>GET</code> request.
```
Comment (ksheurs) [inline source/includes/_resource.analytics.md:88]: follow -> following

File: source/includes/_resource.analytics.md [hunk 4]
```diff
@@ -3,49 +3,454 @@
...
+    <tr class="text-2 border-bottom border--light-gray">
+      <td><strong class="is-block">authentication</strong></td>
+      <td>Captures the change of a logged out -> logged in customer.</td>
+    </td>
+    </tr>  
+  </tbody>
+</table>
+
+<section class="text-2 contain is-internal margin-bottom-medium padding-top-large">
+In addition, platform usage triggered events should be sent with the follow properties as query parameters in the <code>GET</code> request.
```
Comment (ksheurs) [inline source/includes/_resource.analytics.md:320]: follow -> following

**Comment by scottusrobus** [comment]
Thanks @ksheurs will make these changes!

**Comment by scottusrobus** [comment]
@ksheurs fixes deployed.

**Pull request merged**